.travis/docker-build.sh

```diff
@@ -1,17 +1,32 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
 /liqd/aula/.travis/docker-link-stack-workdir.sh
 
 # Change to the source directory which is attacehed as docker volume
 cd /liqd/aula
 
-if [ -z "$1" ] ; then
-    NO_OF_CASES="100"
-else
-    NO_OF_CASES="$1"
-fi
+NO_OF_CASES="100"
+SIZE_OF_CASES="30"
```

Comment:
Not rooted in this PR, but related: why don't we call these QC_MAX_SUCCESS, QC_MAX_SIZE, like on the command line?